srml/contract/src/lib.rs

```diff
@@ -346,7 +384,7 @@ decl_event! {
 		<T as system::Trait>::AccountId,
 		<T as system::Trait>::Hash
 	{
-		/// Transfer happened `from` -> `to` with given `value` as part of a `message-call` or `create`.
+		/// Transfer happened `from` -> `to` with given `value` as part of a `call` or `create`.
```

Comment:
I don't think we should describe implementation in the event doc. Also, `from` and `to` are not part of the parameters here and are out of context.
```suggestion
		/// Transfer happened `from` to `to` with given `value` as part of a `call` or `create`.
```